frontend/src/js/tooltip/Tooltip.tsx

```diff
@@ -177,6 +198,17 @@ const Tooltip = () => {
               />
             )}
           </PinnedLabel>
+          {parent !== label && parent ? (
+            <IndentRoot>
+              <PinnedLabel>
+                <TypeIcon icon={faCaretRight} />
```

Comment:
Hmm, the caret suggests that you can click this to open, no? So I don't think it makes sense to show it here, how about just removing it?